generators/client/templates/src/main/webapp/app/services/auth/_auth.jwt.service.js

```diff
@@ -32,10 +32,31 @@
             var data = {
                 username: credentials.username,
                 password: credentials.password,
-                rememberMe: credentials.rememberMe
```

Comment:
should, because the "data" is actually payload for the uaa. the rememberMe flag is set later in services again